packages/core/workflows-sdk/src/utils/composer/hook.ts

```diff
@@ -107,8 +107,8 @@ export function hook<TOutput>(
     ] as CreateWorkflowComposerContext
   ).hookBinder
 
-  return hookBinder(name, function (context) {
-    return {
+  return hookBinder(name, function () {
+    /*return {
```

Comment:
todo: we can probably rm this file entirely and also update the tests

Reply:
Done